Vue/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "test-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "core-js": "^3.6.5",
+    "devextreme": "^22.1.5",
+    "devextreme-vue": "^22.1.5",
```

Let's update to latest versions:

```suggestion
    "devextreme": "^22.2.4",
    "devextreme-vue": "^22.2.4",
```